### PR TITLE
Background shown for [Continue] button on iOS when it shouldn't overlap the color picker

### LINKED ISSUE #16925: Background shown for [Continue] button on iOS when it shouldn't overlap the color picker
Follow up for https://github.com/status-im/status-mobile/pull/16432

### Steps to Reproduce:
1. Open the app on IOS
2. Go to 'create profile' screen
3. Scroll to make the [Continue] button doesn't overlap the color picker.

### Actual Result:
The [Continue] button has a background shown behind it, even though it doesn't overlap the color picker.

https://github.com/status-im/status-mobile/assets/52490791/1dfea659-bae3-48bf-b68d-e53e12f2a78c

### Expected Result:
Potential solutions:
1. The [Continue] button should not have a background shown behind it if it doesn't overlap the color picker. 
or
2.  Scroll is not available on Create profile screen for IOS devices

CC: @jo-mut

### PR DESCRIPTION
**Pull request by alwx**
Fixes #16925

#### Platforms

- iOS

##### Functional

- new account

status: ready

### CODE REVIEW

File: src/status_im2/contexts/onboarding/create_profile/view.cljs [hunk 1]
```diff
@@ -129,15 +130,21 @@
...
        [rn/scroll-view
         {:on-layout               (fn [event]
                                     (let [height (oops/oget event "nativeEvent.layout.height")]
-                                      (reset! scroll-view-height height)))
+                                      (reset! scroll-view-height height)
+                                      (reset! content-scroll-y 0)))
+         :on-scroll               (fn [event]
+                                    (let [y (oops/oget event "nativeEvent.contentOffset.y")]
+                                      (reset! content-scroll-y y)))
+         :scrollEventThrottle     64
```
Comment (J-Son89) [inline src/status_im2/contexts/onboarding/create_profile/view.cljs]: can be kebab-case? we use 64 elsewhere in the codebase? perhaps this should be a constant somewhere?

**Review by J-Son89 — APPROVED** [review]
perhaps some more details in the pr description would be great but code looks to me 👍 
e.g where/how to test etc

File: src/status_im2/contexts/onboarding/create_profile/view.cljs (hunk 1 above)
Comment (alwx) [reply src/status_im2/contexts/onboarding/create_profile/view.cljs]: we just use 64 :( but also it's different for different lists depending on refresh rate we expect

**Review by jo-mut — APPROVED** [review]

**Comment by qoqobolo** [comment]
@alwx the fix looks good to me, thank you!
But could you rebase the PR, please? I need to rerun tests because they keep failing since this version still contains the crash.
Thanks!

**Comment by alwx** [comment]
@qoqobolo done!

**Comment by qoqobolo** [comment]
@alwx thank you! PR can be merged.

**Pull request merged**